uni/lib/controller/fetchers/exam_fetcher.dart

```diff
@@ -37,8 +37,9 @@ class ExamFetcher implements SessionDependantFetcher {
       for (CourseUnit uc in userUcs) {
         if (!courseExam.examType.contains(
                 '''Exames ao abrigo de estatutos especiais - Port.Est.Especiais''') &&
+            courseExam.examType != 'EE' &&
```

Comment:
Can we verify for both types? Some other faculties or universities may use the old terminology and it won't hurt

Reply:
Both for EE and EAE?